spring-cloud-dataflow-registry/src/main/java/org/springframework/cloud/dataflow/registry/service/AppRegistryService.java

```diff
@@ -72,6 +72,7 @@
 
 	/**
 	 * Deletes all provided {@link AppRegistration}'s.
+	 * @param appRegistrations the {@link AppRegistration}s that should be deleted
```

Comment:
We can avoid saying `should` as these app registrations might not get deleted if they are in use.

Reply:
Thanks.  Will update.